CIP-????/README.md

```diff
@@ -0,0 +1,209 @@
+---
+CIP: XXXX
+Title: A Standard for On-Chain Surveys and Polls
+Category: Metadata
+Status: Proposed
+Authors:
+    - Thomas Lindseth <thomas.lindseth@intersectmbo.org>
+Implementors: []
+Discussions: 'https://github.com/cardano-foundation/CIPs/pull/1107'
```

Comment:
```suggestion
Discussions:
    - https://github.com/cardano-foundation/CIPs/pull/1107
```
Needed to make this parseable & interpreted as a URL in downstream platforms(s).